src/main/java/com/shootoff/camera/cameratypes/PS3EyeCamera.java

```diff
@@ -77,18 +111,28 @@ public static void init() {
 		if (initialized)
 			return;
 
+		try {
+
+			eyecamLib = (eyecam) Native.loadLibrary("eyecam", eyecam.class);
+
+		} catch (UnsatisfiedLinkError exception) {
+
+			initialized = false;
+		}
+
 		eyecamLib.ps3eye_init();
```

Comment:
eyecamLib will be null if there is an UnsatisfiedLinkError.  The function must end before this point if so.